gulp/tasks/test.js

```diff
@@ -200,18 +212,19 @@ function testA11y() {
  * Run PageSpeed Insight tests.
  */
 function testPerf() {
-  _createPSITunnel( function( err, url, tunnel ) {
-    if ( err ) {
-      return plugins.util.log( err );
-    }
+  _createPSITunnel().then( function( params ) {
+    plugins.util.log( 'PSI tests checking URL: http://' + params.url );
```

Comment:
`params.url` is an [array](https://github.com/cfpb/cfgov-refresh/pull/2334/files#diff-a8715b09d86dd13f31e1bb9e3ccc0f71R169) so use `params.url.join( ' ' )` here to get it to display a little nicer.